index.html

```diff
@@ -0,0 +1,44 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Portfolio Project</title>
+    <link rel="stylesheet" href="css/styles.css">
+
+</head>
+
+<body>
+    <main class="container">
+        <header class="main-header">
+            <p class="dev-name">Mehsu Amare</p>
+            <svg width="40" height="40" viewBox="0 0 40 40" fill="none" xmlns="http://www.w3.org/2000/svg">
+                <path fill-rule="evenodd" clip-rule="evenodd"
+                    d="M11 15C11 14.4477 11.4477 14 12 14H28C28.5523 14 29 14.4477 29 15C29 15.5523 28.5523 16 28 16H12C11.4477 16 11 15.5523 11 15ZM11 20C11 19.4477 11.4477 19 12 19L28 19C28.5523 19 29 19.4477 29 20C29 20.5523 28.5523 21 28 21L12 21C11.4477 21 11 20.5523 11 20ZM12 24C11.4477 24 11 24.4477 11 25C11 25.5523 11.4477 26 12 26H28C28.5523 26 29 25.5523 29 25C29 24.4477 28.5523 24 28 24H12Z"
+                    fill="white" />
+            </svg>
```

Comment:
- [ ] The hamburger menu should be wrapped in a `<nav>` tag to improve SEO.

Reply:
- [ ] @melashu great work implementing the changes requested, kindly ensure your hamburger button is wrapped in a `nav` tag as previously requested.

Reply:
- [ ] Kindly ensure your design matches the design given, currently, the background image is missing.Also the font in your paragraph doesn't match the design given, below is a comparison.


Figma Design          |  Your design     
:-------------------------:|:-------------------------:
![image](https://user-images.githubusercontent.com/61408860/176847900-c3153d8c-ad79-45e3-849b-83b191b1a4af.png) |  ![image](https://user-images.githubusercontent.com/61408860/176848287-b98c38ae-c69f-4168-8e2f-bceabdac7101.png)